packages/core/content-manager/admin/src/utils/validation.ts

```diff
@@ -287,7 +302,7 @@ const addMinValidation: ValidationFn =
         (attribute.type === 'component' && attribute.repeatable) ||
         attribute.type === 'dynamiczone'
       ) {
-        if (!attribute.required && 'test' in schema && min) {
+        if (options.status !== 'draft' && !attribute.required && 'test' in schema && min) {
```

Comment:
Should we also handle https://github.com/strapi/strapi/pull/20691/files#diff-c186be7a068bde5fa18b001d2973bfe8a3981eb96e405ef4b87d0a80be5a7dedR330 the same way?

Reply:
Yeah, but let's handle it in another PR, we need to get this merged ASAP to get tests fixed